dev/com.ibm.ws.grpc_fat/fat/src/com/ibm/ws/fat/grpc/ClientConfigTests.java

```diff
@@ -4,7 +4,7 @@
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
- * 
```

Comment:
Update the copyright date range header to include 2023. Also in the classes/files you touch you can remove the "contributors" section at the bottom as it isn't necessary with the EPL-2.0 license any longer.

Reply:
Done.

Reply:
Done